src/components/v2/V2Project/V2ProjectReconfigureModal/index.tsx

```diff
@@ -83,7 +87,11 @@ function ReconfigureButton({
         padding: 10,
         fontWeight: 500,
         borderRadius: radii.sm,
-        border: '1px solid ' + colors.stroke.action.secondary,
+        border:
+          '1px solid ' +
+          (reconfigureHasChanges
+            ? colors.stroke.action.primary
+            : colors.stroke.action.secondary),
```

Comment:
**suggestion** tested locally and i think stroke.secondary might be a lil nicer.

```suggestion
            : colors.stroke.secondary),
```